AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -313,6 +332,17 @@ - (void)setAsyncDelegate:(id<ASCollectionViewDelegate>)asyncDelegate
     super.delegate = (id<UICollectionViewDelegate>)_proxyDelegate;
     _asyncDelegateImplementsInsetSection = ([_asyncDelegate respondsToSelector:@selector(collectionView:layout:insetForSectionAtIndex:)] ? 1 : 0);
   }
+
+  [_flowLayoutInspector cacheSelectorsForCollectionView:self];
```

Comment:
This is a little subtle for my tastes.  It would in fact be a programming error to call [self flowLayoutInspector] here, whereas I was about to recommend we do that to make sure it's allocated.

Is here no applicability of this call point for likely other layoutInspector implementations?  Can we name it something more generic and call it on _layoutDelegate instead?